site/zenodo_rdm/migrator/transform/communities.py

```diff
@@ -113,6 +113,6 @@ def _community_members(self, entry):
 
     def _featured_community(self, entry):
         """Transform the featured community."""
-        if entry["is_featured"]:
+        if entry.get("is_featured", False):
```

Comment:
according to the extraction query isn't the "is_featured" property always there?